_i18n/en.yml

```diff
@@ -112,7 +112,9 @@ faq:
   item18:
     question: Can I install IRMA on Android without Google Play Store?
     answer: Yes, you can download the IRMA app here<span>:</span> <a href="https://f-droid.org/en/packages/org.irmacard.cardemu/">F-droid</a>, or via <a href="https://irma.app/irma.apk" target="_blank">https://irma.app/irma.apk</a>. If you install IRMA this latter way, the app will not recieve updates automatically.
-
+  item19:
+    question: What is my IRMA AppID?
```

Comment:
I'm a bit doubting whether it should be your App ID, your app ID or your AppID, and in dutch maybe App-ID or app-ID (applicatie-identifier). It all seems weird in some way...

Reply:
I suggest to keep it as close to what is normal in each language as possible:

* en: "app ID"
* nl: "app-ID"